ci/cirrus_wheels.yml

```diff
@@ -29,6 +29,8 @@ cirrus_wheels_linux_aarch64_task:
         CIBW_BUILD: cp311-manylinux* cp312-manylinux*
   env:
     CIBW_PRERELEASE_PYTHONS: False
+    CIBW_ENVIRONMENT: >
+      PKG_CONFIG_PATH=/project
```

Comment:
I think in `main` we still have the pre-release NumPy stuff here, but when resolving the merge conflict I assumed that this is the one extra thing I needed to keep in this file. I guess we'll find out for sure when we probe the wheel builds here.